images/base-glibc-busybox-bash/Dockerfile

```diff
@@ -43,13 +43,16 @@ RUN mkdir -p \
 # Remove glibc files. They are incomplete and get substituted by `install-pkgs`.
 RUN rm -rf ./lib ./lib64
 
+# Install helper tools used by install-pkgs.
+RUN apt-get update -qq \
+    && \
+    apt-get install --yes --no-install-recommends \
```

Comment:
should we remove this again after using?

Reply:
It's just in the build container but not in the final one, so we don't really have to care :shrug: ;).

Reply:
Ah true, wrong file. +1